### PR TITLE
Fix kibana rbac and calculate minimum es master replicas

### PR DESCRIPTION
**Pull request by alvaroaleman**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Documentation**:

**Does this PR introduce a user-facing change?**:

```release-note
none
```

### CODE REVIEW

File: config/logging/elasticsearch/templates/es-data-stateful.yaml [hunk 1]
```diff
@@ -85,10 +85,8 @@ spec:
           valueFrom:
             resourceFieldRef:
               resource: limits.cpu
-        {{- range $key, $value := .Values.logging.elasticsearch.cluster.env }}
-        - name: {{ $key }}
-          value: {{ $value | quote }}
-        {{- end }}
+        - name: MINIMUM_MASTER_NODES
+          value: "{{ add (div .Values.logging.elasticsearch.master.replicas 2|int64) 1 }}"
```
Comment (alvaroaleman) [inline config/logging/elasticsearch/templates/es-data-stateful.yaml:89]: This is needed to make the thing work with an uneven number of replicas.

The underlying arthmetic func already use ints, but I'd like to not rely on implementation details

Comment (xrstf) [reply config/logging/elasticsearch/templates/es-data-stateful.yaml:89]: Why did you remove the ability to configure ES via environment variables? I would have guessed you'd just add/overwrite the `MINIMUM_MASTER_NODES` variable.

Comment (alvaroaleman) [reply config/logging/elasticsearch/templates/es-data-stateful.yaml:89]: Yeah I can edit it to just do that

**Comment by xrstf** [comment]
/hold

Can you please also bump the chart versions?

**Comment by alvaroaleman** [comment]
@xrstf Done

**Pull request merged**